packages/backend/src/api/job/jobService.ts

```diff
@@ -1,13 +1,17 @@
-import { eq } from 'drizzle-orm';
 import { StatusCodes } from 'http-status-codes';
 import { ResponseStatus, ServiceResponse } from 'src/common/models/serviceResponse';
-import { job, jobCategory, jobLevel } from 'src/db/schema';
 import { logger } from 'src/server';
+import { ZodError } from 'zod';
 
-import { db } from '../../db';
-import { Job, JobSchema, SavedJob } from './jobModel';
+import { Job, SavedJob } from './jobModel';
 import { jobRepository } from './jobRepository';
-import { CreateJobRequest, PutJobRequest } from './jobRequest';
+import {
+  CreateJobCategorySchema,
+  CreateJobLevelSchema,
+  CreateJobRequest,
+  JobIDSchema,
```

Comment:
i cant find JobIdSchema

Reply:
sorry I forgot to upload the new file I will now